addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -66,23 +72,23 @@ func build_gamepack_text() -> String:
 		texturetypes_str += texture_type
 		if texture_type != texture_types[-1]:
 			texturetypes_str += " "
-	
+
```

Comment:
It's really hard to track the actual changes when you've (I assume accidentally?) deleted every indentation that doesn't have any code on the same line. Is there any chance you could undo that and avoid it in the future?

Reply:
Oh shoot, that's my Godot editor auto clearing whitespace. Will fix this for the future + this commit as well

Reply:
The main reason I don't clean up white space indentations in GDScript is that it makes it easier to track what lines of code belong to which code blocks when they're spaced apart with line breaks when in the Godot's script editor. Makes it easier to parse when you're dealing with extremely long scripts (like func_godot_map.gd). I'm honestly not sure of any good reason to clean up white space indentations to be honest, since there isn't a measurable impact on performance.

That doesn't mean white space _spaces_ shouldn't be cleaned up, but that's also not really a rampant problem as far as I've seen.